docs/.vitepress/config.ts

```
@@ -0,0 +1,135 @@
+		lastUpdated: {
+			text: 'Updated at',
+			formatOptions: {
+				dateStyle: 'full',
+				timeStyle: 'medium',
+			},
+		},
+	},
+
+	vite: {
+		plugins: [
+			groupIconVitePlugin(),
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)